Lib/pathlib/__init__.py

```diff
@@ -11,6 +11,7 @@
 import os
 import posixpath
 import sys
+import warnings
```

Comment:
Does this meaningfully affect import times? We've made a somwhat concerted effort to reduce them recently.

Reply:
Import is now delayed :-)